test-cases/upgrades/rolling-upgrade.yaml

```diff
@@ -1,6 +1,6 @@
 # TODO: need to qualify on GCE and AWS
 
-test_duration: 250
+test_duration: 500
```

Comment:
i think this is related to the 1st comment, but why the test duration is being doubled? because of Gemini?